packages/vuetify/src/components/VCheckbox/VCheckbox.tsx

```diff
@@ -48,6 +49,10 @@ export const VCheckbox = defineComponent({
     ...makeValidationProps(),
   },
 
+  emits: {
+    'update:modelValue': (val: any) => true,
```

Comment:
is there a reason for using `val: any` instead of `val: unknown`?

Reply:
Had some problems with unknown here in another component.